### ISSUE TITLE
Add a forward button

### ISSUE THREAD

**Issue by aaronfranke** [browsh-org/browsh#257]
I [noticed](https://www.brow.sh/docs/keybindings/) that there is a back button, but no forward button. It would nice to have a forward button.

Should apply to all systems so I'm not including my system information.

**Comment by tombh** [comment]
I agree about this. Though I'm not sure how best to approach it. How would you feel if it was just a shortcut? The thing is that I don't want to get into being able to give users the ability to manually configure the UI. Hang on, but it's harldy a big thing is it! It's a little button, ok, ok, maybe we can just add it :D
